capstone/src/main/webapp/club-registration.html

```diff
@@ -18,13 +18,17 @@ <h2>Club Registration</h2>
         <label for="name">Description:</label>
         <input type="text" name="description" id="club-description"/>
       </div>
+      <div class="form-group">
+        <label for="name">Website:</label>
```

Comment:
These "for" attribute values should match the id attribute of the related input field